Doc/library/stdtypes.rst

```diff
@@ -2760,12 +2762,14 @@ arbitrary binary data.
             bytearray.decode(encoding="utf-8", errors="strict")
 
    Return a string decoded from the given bytes.  Default encoding is
-   ``'utf-8'``. *errors* may be given to set a different
-   error handling scheme.  The default for *errors* is ``'strict'``, meaning
-   that encoding errors raise a :exc:`UnicodeError`.  Other possible values are
-   ``'ignore'``, ``'replace'`` and any other name registered via
-   :func:`codecs.register_error`, see section :ref:`error-handlers`. For a
-   list of possible encodings, see section :ref:`standard-encodings`.
+   ``'utf-8'``.  And for a list of possible encodings, see section
+   :ref:`standard-encodings`.
+
+   *errors* may be given to set a different error handling scheme.  The
+   default for *errors* is ``'strict'``, meaning that encoding errors raise a
+   :exc:`UnicodeError`.  Other possible values are ``'ignore'``, ``'replace'``
+   and any other name registered via :func:`codecs.register_error`, see
+   section :ref:`error-handlers`.
```

Comment:
```suggestion
   *errors* controls how decoding errors are handled.
   If ``'strict'`` (the default), a :exc:`UnicodeError` exception is raised.
   Other possible values are ``'ignore'``, ``'replace'``,
   and any other name registered via :func:`codecs.register_error`.
   See :ref:`error-handlers` for details.
```

* Fix "encoding errors" -> "decoding errors"

As above:
* Be more direct, descriptive and conventional with summary line
* Be more concise overall and avoid repetition
* Avoid grammar error (comma splice)